superset-frontend/src/SqlLab/components/ScheduleQueryButton.tsx

```diff
@@ -0,0 +1,214 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { FunctionComponent, useState } from 'react';
+import Form, { FormProps, FormValidation } from 'react-jsonschema-form';
+import { Col, FormControl, FormGroup, Row } from 'react-bootstrap';
+import { t, styled } from '@superset-ui/core';
+import * as chrono from 'chrono-node';
```

Comment:
@AAfghahi This library is missing type definition and local webpack is throwing TS errors. I'm surprised CI didn't catch this.

Reply:
Hey! I was running into the same issue, did you do npm install? That solved it for me. The earlier versions don't have type definitions and the later ones do.

Reply:
I'm pretty sure I did but I was still seeing the error. Nevertheless, I bumped it again to the latest in `package.json` and it works.

Reply:
Thank you for bumping the version.